nautobot/dcim/api/urls.py

```diff
@@ -89,6 +89,8 @@
 router.register("virtual-device-contexts", views.VirtualDeviceContextViewSet)
 router.register("interface-vdc-assignments", views.InterfaceVDCAssignmentViewSet)
 
+# Device Cluster Assignments
+router.register("device-cluster-assignments", views.DeviceClusterAssignmentViewSet)
```

Comment:
We should alpha-sort this file at some point, but this placement is fine for now.